src/bn.js

```diff
@@ -0,0 +1,7 @@
+const Web3 = require('web3');
+
+const BN = Web3.utils.BN;
+
+module.exports = (value) => {
+  return new BN(value);
```

Comment:
value should be cast to string to protect from assertion error or use of w3.utils.toBN(value).